packages/service-library/tests/test_archiving_utils_extra.py

```diff
@@ -13,8 +13,11 @@
     unarchive_dir,
 )
 
+
 from .test_utils import print_tree
 
+# FIXTURES
```

Comment:
I tend to remove these ```# FIXTURES``` ```# TESTS``` comments actually... cause sometimes they are even wrong since if isort moves some imports and there is a comment around it moves with it. and also ```@pytest.fixture``` kinds of self explain it's a fixture.

Reply:
@sanderegg added in #3348